corehq/apps/data_cleaning/models.py

```diff
@@ -257,3 +259,28 @@ class BulkEditChange(models.Model):
 
     class Meta:
         ordering = ["created_on"]
+
+    def edited_value(self, case):
+        simple_transformations = {
+            EditActionType.REPLACE: lambda x: self.replace_string,
+            EditActionType.FIND_REPLACE: lambda x: x.replace(self.find_string, self.replace_string),
+            EditActionType.STRIP: str.strip,
+            EditActionType.TITLE_CASE: str.title,
+            EditActionType.UPPER_CASE: str.upper,
+            EditActionType.LOWER_CASE: str.lower,
+            EditActionType.MAKE_EMPTY: lambda x: "",
+            EditActionType.MAKE_NULL: lambda x: None,
+        }
+
+        if self.action_type in simple_transformations:
+            old_value = case.get_case_property(self.property)
+            return simple_transformations[self.action_type](old_value)
+
+        if self.action_type == EditActionType.COPY_REPLACE:
+            old_value = case.get_case_property(self.copy_from_property)
+            return old_value.replace(self.find_string, self.replace_string)
+
+        if self.action_type == EditActionType.RESET:
+            raise UnsupportedActionException(f"{EditActionType.RESET} is not applied by calling edited_value")
```

Comment:
this should just return the value of the current case property value. I think this should be implemented.

Reply:
The intention is to reset all previous edits? Based on the spec, I thought it only reset the previous action.

Reply:
no reset all edits, not the previous action. It's the equivalent of hitting the "x" in the column or "Undo all edits" in the menu for the action
![Screenshot 2025-02-19 at 10 29 15 PM](https://github.com/user-attachments/assets/85e9d32c-d289-41b5-89db-ee966c85e8be)

Reply:
Then why would it go back to the original value of the case property, when it may have had additional edits?

I'm not addressing reset right now.